cellx/networks/classifier.py

```diff
@@ -1 +1,13 @@
-# this will be the classifier
+# Building the CNN classifier:
+
+import tensorflow as tf
+from tensorflow import keras as K
+
+from cellx import layers
+
+input = K.Input(shape=(2,))
+encoder = layers.Encoder2D()(input)
```

Comment:
Might perhaps be worth specifying the encoder layer when the build_classifier function is called as well, e.g.
`def build_classifier(shape: tuple, outputs: int, dense_filters: int = 512):
    """add docstrings :)"""
    input = K.Input(shape=shape)
    ...
    return classifier_model`

seeing as it can be helpful to experiment with various instances of Encoder2D (with different numbers of layers, units per layer etc) and that would be served I think by being able to call Encoder2D outside of build_classifier and then incorporate it in after.